src/games/strategy/triplea/ai/proAI/logging/ProLogUI.java

```diff
@@ -10,7 +10,8 @@
  * Class to manage log window display.
  */
 public class ProLogUI {
-  private static TripleAFrame s_frame = null;
+  @SuppressWarnings("unused")
```

Comment:
Hmm, I don't think just adding a suppress warning annotation is the right way to fix this. If the variable is not used, it can likely be removed. But be careful since some are used by reflection and are actually used.. Generally the stuff in remote and/or delegate code, or that has to do with XML parsing is potentially used by reflection. Testing though is the only way to know in the end, connect to lobby, play a game, and make sure you do not get a variable not found error.